src/graph/context/Iterator.cpp

```diff
@@ -439,11 +439,8 @@ Value GetNeighborsIter::getVertex(const std::string& name) const {
   if (!valid()) {
     return Value::kNullValue;
   }
-
   auto vidVal = getColumn(0);
-  if (UNLIKELY(!SchemaUtil::isValidVid(vidVal))) {
```

Comment:
Why remove this?

Reply:
the type of vid has been verified at the storage layer